internal/service/securitylake/aws_log_source.go

```diff
@@ -172,13 +155,13 @@ func (r *awsLogSourceResource) Read(ctx context.Context, request resource.ReadRe
 	}
 
 	// We can't use AutoFlEx with the top-level resource model because the API structure uses Go interfaces.
-	var sourceData awsLogSourceSourceModel
+	var sourceData awsLogSourceResourceModel
 	response.Diagnostics.Append(fwflex.Flatten(ctx, logSource, &sourceData)...)
 	if response.Diagnostics.HasError() {
 		return
 	}
 
-	data.Source = fwtypes.NewListNestedObjectValueOfPtrMust(ctx, &sourceData)
+	data = sourceData
```

Comment:
I think this causes the existing `id` value to be lost and hence `ImportStateVerify: New resource missing identifier attribute "id", ensure attribute value is properly set or use ImportStateVerifyIdentifierAttribute to choose different attribute` acceptance test errors.